pontoon/teams/views.py

```diff
@@ -136,6 +138,7 @@ def ajax_projects(request, locale):
         {
             "locale": locale,
             "projects": projects,
+            "project_stats": Project.objects.all().stats_data(locale),
```

Comment:
Let's limit the list of projects to gather stats for:

```suggestion
            "project_stats": Project.objects.visible().stats_data(locale),
```